README.md

```diff
@@ -466,20 +466,22 @@ This Example would create three radio Buttons with the labels: Option A, Option
 | keyword | mandatory | fallback value | description                                                                                                                                                                                                                                                                             |
 |---------|-----------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | type    | yes       | -              | widget type definition: **slider**                                                                                                                                                                                                                                                   |
-| label   | no        | Wert           | headline of the slider widget                                                                                                                                                                                                                                                            |
+| name    | no        | Wert           | headline of the slider widget                                                                                                                                                                                                                                                            |
 | start   | no        | 0.0            | start value of the slider                                                                                                                                                                                                                                                            |
 | end     | no        | 1.0            | end value of the slider                                                                                                                                                                                                                                                            |
 | step    | no        | 0.1            | step value of the slider
 | unit    | no        | -              | unit that will be displayed after the slider value
+| label   | no        | relativerangefilter_duration | defines the filter that will be applied

```

Comment:
Hab hier kurz gestockt, weil label dafür sehr verwirrend klingt, auch wenn es technisch gesehen ja ein label an den params ist. Hab aber gesehen, das es bei den Termfiltern oder so auch so ist und der name ist dann das label. Also immerhin gleich verwirrend überall jetzt ^^ (immerhin kann man hier einen sinnvollen default geben) also alles gut, kann man sich zu gegebener Zeit für alle etwas überlegen.

Reply:
Ja das ist, naming ist schwer.
eigentlich ist das label in dem Zusammenhang eine Art Command bzw Filtertyp